samples/browser/src/main/java/org/mozilla/samples/browser/addons/WebExtensionActionPopupActivity.kt

```diff
@@ -0,0 +1,95 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.samples.browser.addons
+
+import android.content.Context
+import android.os.Bundle
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import kotlinx.android.synthetic.main.fragment_add_on_settings.*
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import mozilla.components.browser.state.action.WebExtensionAction
+import mozilla.components.concept.engine.EngineSession
+import mozilla.components.concept.engine.EngineView
+import mozilla.components.lib.state.ext.consumeFrom
+import org.mozilla.samples.browser.R
+import org.mozilla.samples.browser.ext.components
+
+/**
+ * An activity to show the pop up action of a web extension.
+ */
+class WebExtensionActionPopupActivity : AppCompatActivity() {
```

Comment:
We also need to set the title of the popup to the name of the extension (and icon, but could skip that for now). Can we do that, see UX mock:
<img width="229" alt="Screen Shot 2020-01-21 at 3 56 51 PM" src="https://user-images.githubusercontent.com/472523/72842677-d7054900-3c66-11ea-8ef3-49c03ae5ff5c.png">